presto-hive-common/pom.xml

```diff
@@ -95,6 +105,11 @@
             <artifactId>drift-api</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
I'd prefer not to add this dependency to common code. Anyway it can be isolated to just the aws package?

Reply:
I just realised that I am only using one class from this dependency in the class com.facebook.presto.hive.aws.security.s3.AWSS3SecurityMapping. 

It is being used to form the credentials object here: https://github.com/prestodb/presto/pull/21622/files#diff-33e654b04e220ef6b736fee85d4cf10f15e0d5255c6af614ce8d4a2000db2b27R51. 

We have a couple of options on how we can exclude the aws dependency from hive-common:

1. Make a clone(or similar class) of the BasicAWSCredentials class within Presto and use that here
2. Rather than creating the credentials object here, modify the current implementation and keep the access key and secret key as is in this class and, create the credentials object when required.

CC: @tdcmeehan

Reply:
Option 1 sounds fine to me, because we just need the key and secret key, and those can just be opaque strings.